homeassistant/components/openuv/sensor.py

```diff
@@ -43,17 +42,52 @@
 UV_LEVEL_MODERATE = "Moderate"
 UV_LEVEL_LOW = "Low"
 
+SENSORS = {
```

Comment:
All the constants maybe into `const.py` inside integration?

Reply:
I traditionally haven't done this; instead, I've followed the pattern used by HASS itself (centralize a constant into `const.py` if it is used in more than one place, otherwise define it where it is used). Should I not be doing that?